### ISSUE TITLE
bleno in the browser

### ISSUE THREAD

**Issue by omizrahi99** [sandeepmistry/bleno#463]
Would it be possible to use bleno in client-side JavaScript to make the browser act as a peripheral?

**Comment by monteslu** [comment]
nope.  The Web BLE spec is only for bluetooth clients.  There's isn't a web API for creating GATT services
